packages/docs-site/src/content/docs/docs/plugins/bitwarden.mdx

```diff
@@ -0,0 +1,144 @@
+---
+title: Bitwarden plugin
+description: DMNO's Bitwarden plugin allows you to securely access your secrets stored in Bitwarden Secrets Manager.
+npmPackage: "@dmno/bitwarden-plugin"
+---
+
+import { Steps, Icon } from '@astrojs/starlight/components';
+import TabbedCode from '@/components/TabbedCode.astro';
+
+This DMNO plugin allows you to securely access your secrets stored in [Bitwarden Secrets Manager](https://bitwarden.com/products/secrets-manager/). Please note that this plugin is **not compatible with Bitwarden's Password Manager product**. Authentication with Bitwarden uses [Machine Account Access Tokens](https://bitwarden.com/help/access-tokens/).
+
+## Installation & setup
+
+Install the package in the service(s) that will use secrets from Bitwarden.
+
+<TabbedCode packageName="@dmno/bitwarden-plugin" />
+
+-----
+
+After installation, you'll need to initialize the plugin in your dmno config and add a config item to hold your machine account access token. You can explicitly wire the plugin up to the service account token if using multiple tokens at once, or it will be injected by default. It's ok if you have not created the machine account or access token - we'll do that in the next section.
```

Comment:
```suggestion
After installation, you'll need to initialize the plugin in your `config.mts` and add a config item to hold your machine account access token. You can explicitly wire the plugin up to the service account token if using multiple tokens at once, or it will be injected by default based on the `BitwardenSecretsManagerTypes.machineAccountAccessToken` type. It's ok if you have not created the machine account or access token - we'll do that in the next section.
```